web-js/4.x/add-attachments-with-42/README.md

```diff
@@ -0,0 +1,85 @@
+#Add Attachments To FeatureService with JavaScript 4.2
+
+##About
+FeatureServices can be configured to allow files to be attached to individual features in the service. In the 3.x JavaScript API methods to retrieve, add and delete attachments were provided as part of the FeatureLayer class. Additionally widgets such as the attachment and attribute inspector allowed attachments to be viewed and edited. In the 4.x API no widgets or methods have been provided to work with attachments at this time. However it is still possible to directly query and edit attachments using the ArcGIS REST API. This sample shows how to work with attachments within a 4.x application.
```

Comment:
language like *4.x* and *'at this time'* will become stale.  if would be better to just consistently refer to *4.2*.